elasticdl/python/common/model_utils.py

```diff
@@ -191,6 +191,31 @@ def get_model_spec(
     )
 
 
+def get_training_func_spec(
+    model_zoo, model_def, feed, custom_data_reader,
+):
+    """Get the model spec items in a tuple.
+
+    The model spec tuple contains the following items in order:
+
+    * The `training_func` of training loop.
```

Comment:
What does `training_func` mean? `forward` method or the function contains the training loop?

Reply:
The function contains the training loop.